src/pipx/main.py

```diff
@@ -183,6 +183,18 @@ def package_is_url(package: str, raise_error: bool = True) -> bool:
     return False
 
 
+def package_is_abspath(package: str):
+    if Path(package).is_absolute() or Path(package).exists():
+        raise PipxError(
+            pipx_wrap(
+                f"""
+                Error: Path '{package}' given as package.
+                Expected the name of an installed package.
+                """
+            )
+        )
```

Comment:
Why `is_absolute`? Does relative paths (e.g. `foo/bar`) not count here? The error message doesn’t match the check logic.